packages/ts-transformers/src/internal/decorators/property.ts

```diff
@@ -58,11 +59,41 @@ export class PropertyVisitor implements MemberDecoratorVisitor {
     }
     const options = this._augmentOptions(arg0);
     const name = property.name.text;
-    litClassContext.litFileContext.nodeReplacements.set(property, undefined);
+    const factory = this._factory;
+
+    if (isGetter && property.decorators) {
+      // Filter out the current decorator
+      let decorators: ts.Decorator[] | undefined = property.decorators.filter(
+        (dec) => dec !== decorator
+      );
+
+      // If there are no decorators prevent the tslib package from being
+      // imported by unassigning the decorators array.
+      if (decorators.length === 0) {
+        decorators = undefined;
+      }
+
+      // Decorators is readonly so clone the property.
+      const getterWithoutDecorators = factory.createGetAccessorDeclaration(
```

Comment:
Can we also test that comments are preserved on this get accessor?

Should only require these additional lines:

```ts
const nonBlankLineSyntheticComments = ts
  .getSyntheticLeadingComments(property)
  ?.filter((comment) => comment.text !== BLANK_LINE_PLACEHOLDER_COMMENT);
ts.setSyntheticLeadingComments(
  getterWithoutDecorators,
  nonBlankLineSyntheticComments
);
```